Packages/Tracking Preview/UI Input/Runtime/Scripts/PointerElement.cs

```diff
@@ -23,7 +23,8 @@ public class PointerElement : MonoBehaviour
         #region Properties
 
         public event Action<PointerElement, Hand> OnPointerStateChanged;
-        public event Action<bool> On;
+
+        //public event Action<bool> On;
```

Comment:
Event is never invoked or subscribed to.

Reply:
since it's never invoked, is this maybe just a bug where that implementation was accidentally removed?

Reply:
@Craig-J Please see above

Reply:
Would you like to add the functionality to actually raise this event?

atm nothing raises the event anyway, so we can either remove it, or finish it.